.github/docker/Dockerfile.centreon-web-bullseye

```diff
@@ -0,0 +1,130 @@
+FROM centreon.jfrog.io/docker/debian:bullseye AS web_fresh
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN <<EOF
```

Comment:
Le Heredoc Dockerfile, ca a l'air d'être une spécificité "récente" de BuildKit.
Est-ce bien supporté partout ? buildx etc ...

Reply:
Yes, we always use docker with buildkit in the github workflows